tasks/copy_to_glacier/README.md

```diff
@@ -241,8 +241,8 @@ These settings can often be derived from the collection configuration in Cumulus
         "cma": {
           "event.$": "$",
           "task_config": {
+            "s3MultipartChunksizeMb": "{$.meta.collection.s3MultipartChunksizeMb"},
```

Comment:
Shouldn't the `"{$.meta.collection.s3MultipartChunksizeMb"}` value be `"{$.meta.collection.meta.s3MultipartChunksizeMb}"`? Per the documentation.

Reply:
Yup. Fixed.